### ISSUE TITLE
Limited Stock of Airframes (with logistics to resupply)

### ISSUE THREAD

**Issue by Igneous01** [KaukausInsurgency/kaukasusinsurgency-dcs#198]
It would be cool to support limited numbers of airframes (whether this is per airfield/farp or global is to be decided).

So for example, configure that only 8 AV8Bs can be slotted into, if they are destroyed no one can slot into an AV8B. Then implement logistic convoys to resupply an airfield to regain those AV8B airframes so players can slot in again.

**Comment by thebgpikester** [comment]
And the same for enemy? But there's a catch. The last thing you want to do is exclude players from a game. This is right up against the balance of gameplay and something realistic. It also overlaps the lives system which does the same, perhaps even better. Regaining lives per hour gives a good effect to pilots watching their carelessness.

I'd like to see a global resupply system that affected ground units for sure, but that's a different topic.

**Comment by Igneous01** [comment]
Yeah this was something two other users mentioned to me as an ask (they both saw how it worked in dynamicdcs, and thought it would be cool to have here). From their perspective if you have lives tiered by airframe you encourage players to use more diverse airframes (f-5-e instead of f15, or mig-21 instead of su-27). But this assumes that players are knowledgeable in those airframes... If they're not then they are excluded in a way from playing the game.
